coverage.sh

```
@@ -0,0 +1,291 @@
+            ;;
+        *)
+            print_warning "Unknown argument: $arg"
+            ;;
+    esac
+done
+
+# Create coverage output directory
+mkdir -p "$COVERAGE_DIR"
+
+# Change to mvp_site directory
+cd mvp_site
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)